platformcomponents/desktop/attachment-file-error.json

```diff
@@ -0,0 +1,16 @@
+{
+  "attachment-file-error": {
+    "sent": {
+      "comment": "Applied to sent & blocked file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5533%3A11688",
+      "background": "@theme-background-primary-disabled",
```

Comment:
I do not find this background, where is it?

Reply:
**Nice catch!**
Indeed, there is no such background used, it was from a conversation I had with Morgan Bathe.
I have replaced the wrong `@theme-background-primary-disabled` with `@theme-background-secondary-normal`.
Also added colors for `#hovered` and `#active` states, in this commit:
[Corrected background with states for attachment-file-error](https://github.com/momentum-design/tokens/pull/174/commits/b3ff41a74d7ff241df4c3c5ea73b321e9dd02b0a)
Changing this conversation to Resolved.